ui/component/previewOverlayProperties/view.jsx

```diff
@@ -99,7 +104,8 @@ export default function PreviewOverlayProperties(props: Props) {
           )}
           {isCollection && claim && !iconOnly && <div>{claimLength}</div>}
           {!iconOnly && isStream && <VideoDuration uri={uri} />}
-          {isStream && <FileType uri={uri} small={small} />}
+          {isStream && !isLivestream && <FileType uri={uri} small={small} />}
+          {isLivestream && <Icon icon={ICONS.LIVESTREAM_MONOCHROME} size={13} />}
```

Comment:
Is the icon for the Upcoming one smaller?  Even in Desktop, it feels tiny-er than the rest

<img width="281" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/22c5dcf8-822d-4f9a-8efe-1a47b3fc1a60">

Reply:
Should be exactly the same size like in my screenshot above unless I forgot a commit.

Reply:
I always rebase to latest `master`.  Maybe a clash with the style patch?